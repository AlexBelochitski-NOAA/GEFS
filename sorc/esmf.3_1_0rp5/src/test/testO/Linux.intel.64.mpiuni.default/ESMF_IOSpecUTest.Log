20190610 112231.557787 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112231.558043 INFO     PET0  Beginning Test, file ESMF_IOSpecUTest.F90, line          66
20190610 112231.558061 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112231.558083 INFO     PET0  PASS  IOSpec Set Test, ESMF_IOSpecUTest.F90, line          77
20190610 112231.558091 INFO     PET0  PASS  Get IOSpec File Name Test, ESMF_IOSpecUTest.F90, line          85
20190610 112231.558099 INFO     PET0  PASS  Verify IOSpec Filename Test, ESMF_IOSpecUTest.F90, line          92
20190610 112231.560288 INFO     PET0  PASS  Get IOSpec File Format Test, ESMF_IOSpecUTest.F90, line         101
20190610 112231.560297 INFO     PET0  PASS  Verify IOSpec File Format  Test, ESMF_IOSpecUTest.F90, line         108
20190610 112231.560314 INFO     PET0  Number of failed tests:           0
20190610 112231.560320 INFO     PET0  Ending Test, file ESMF_IOSpecUTest.F90, line         118
