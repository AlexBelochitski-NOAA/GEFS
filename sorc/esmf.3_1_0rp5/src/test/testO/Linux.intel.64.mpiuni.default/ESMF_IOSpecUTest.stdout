  Beginning Test, file ESMF_IOSpecUTest.F90, line          66
  NUMBER_OF_PROCESSORS           1
  PASS  IOSpec Set Test, ESMF_IOSpecUTest.F90, line          77
  PASS  Get IOSpec File Name Test, ESMF_IOSpecUTest.F90, line          85
  PASS  Verify IOSpec Filename Test, ESMF_IOSpecUTest.F90, line          92
  File name = 
 testFilename                                                                   
                                                  
  PASS  Get IOSpec File Format Test, ESMF_IOSpecUTest.F90, line         101
  PASS  Verify IOSpec File Format  Test, ESMF_IOSpecUTest.F90, line         108
  Number of failed tests:           0
  Ending Test, file ESMF_IOSpecUTest.F90, line         118
